libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -237,7 +237,8 @@ void AP_OSD::update_current_screen()
             if (switch_debouncer) {
                 next_screen();
                 previous_channel_value = channel_value;
-            } else {
+            }
```

Comment:
This indentation style is strange. Are you sure you picked the right indentation style?

Reply:
I ran the style checker command line exactly as the docs stated to do....dont know how to pick a different style